libr/cons/grep.c

```diff
@@ -378,6 +378,25 @@ R_API void r_cons_grep_parsecmd(char *cmd, const char *quotestr) {
 	}
 }
 
+R_API char * r_cons_grep_strip(char *cmd, const char *quotestr) {
+	char *ptr = NULL;
+
+	if (!cmd) {
+		return NULL;
+	}
+
+	ptr = preprocess_filter_expr(cmd, quotestr);
```

Comment:
missing space before (.

You can negate this conditional and put everything inside if (cmd) { ptr=...; chop(); } return ptr;